dandi/cli/cmd_delete.py

```diff
@@ -5,11 +5,16 @@
 
 @click.command()
 @click.option("--skip-missing", is_flag=True)
+@click.option(
+    "--force",
```

Comment:
This option should also be listed in `docs/source/cmdline/delete.rst`.

Reply:
Previously in https://github.com/dandi/dandi-cli/pull/831 sphinx-cli ... formalized that in a new  https://github.com/dandi/dandi-cli/issues/1002